### ISSUE TITLE
KEEP unimputed issuse

### ISSUE THREAD

**Issue by online-translation** [statgen/Minimac4#7]
Hi, I has a chipped data, some of variants was not covered by the imputation panel, but I want to keep all of them into the imputed result, does minimac proivides such function, if yes, then which parameters should i use?

Thanks a lot.

**Comment by Santy-8128** [comment]
Yes, on minimac3/4 you can use --allTypedSites to have all variants in the
final output file.

*Regards,*
*Sayantan Das,*



*Post-Doctoral Fellow Department of BiostatisticsUniversity of Michigan,
Ann Arbor.MI 48109-2029*
